Automattic/wp-calypso
# Global Styles: Add error handling to avoid the page crash

Related to p1691576959289179-slack-C02FMH4G8

## Proposed Changes

* Made `@automattic/calypso-sentry` package so we can send messages to sentry outside the client scope
* Wrap `useGlobalStylesOutput` hook to catch any internal error and send logs to sentry

## Testing Instructions



* Revert D118678-code on your sandbox, and disable the cache
* Go to Theme Showcase
* Search for "Zaino"
* Go to the Theme Details Page of the “Zaino” theme
* Ensure the page won't crash

## Pre-merge Checklist
